### ISSUE TITLE
Is it able to chain pending transaction?

### ISSUE THREAD

**Issue by anhcao142** [StardustCollective/dag4.js#3]
Hi, is it able to chain pending or future transactions? For example, I want to send Dag from my address to 3 different addresses, I don't want to wait for each transaction to finalize before creating a new one, can I create 3 transactions at the same time which the next tx chaining from the previous and submit all of them to the network?

**Comment by ffox77** [comment]
Yes. You chain them by providing the hash of the previous tx and increasing the ordinal. You can use the method dag4.keyStore.generateTransaction and pass it these in the lastRef parameter. I can create an example of this in the demo rep later.

**Comment by anhcao142** [comment]
Thanks @ffox77 , I understand the method, I'll update if it's not working
